airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -81,88 +81,42 @@ public ConnectionManagerWorkflowImpl() {}
 
   @Override
   public void run(final ConnectionUpdaterInput connectionUpdaterInput) throws RetryableException {
-    connectionId = connectionUpdaterInput.getConnectionId();
     try {
-      if (connectionUpdaterInput.getWorkflowState() != null) {
-        workflowState = connectionUpdaterInput.getWorkflowState();
-      }
       try {
         syncWorkflowCancellationScope = Workflow.newCancellationScope(() -> {
```

Comment:
can we rename `syncWorkflowCancellationScope` to `syncWorkflow`? I understand why the original name was called, however my initial read of it is this is defined cancellation behaviour, and I had to read the constructor to understand what is going on.

Reply:
I think it would be easier to read if we pulled out the anonymous block into a function to give us this:

```
@Override
  public void run(final ConnectionUpdaterInput connectionUpdaterInput) throws RetryableException {
    try {
      try {
        generateSyncWorkflow(connectionUpdaterInput);
        syncWorkflow.run();
      } catch (final CanceledFailure cf) {
        // When a scope is cancelled temporal will thow a CanceledFailure as you can see here:
        // https://github.com/temporalio/sdk-java/blob/master/temporal-sdk/src/main/java/io/temporal/workflow/CancellationScope.java#L72
        // The naming is very misleading, it is not a failure but the expected behavior...
      }

      if (workflowState.isDeleted()) {
        deleteConnectionBeforeTerminatingTheWorkflow();
        return;
      }

      if (workflowState.isCancelledForReset()) {
        reportCancelledAndContinueWith(true, connectionUpdaterInput);
      }

      if (workflowState.isCancelled()) {
        reportCancelledAndContinueWith(false, connectionUpdaterInput);
      }

    } catch (final Exception e) {
      log.error("The connection update workflow has failed, will create a new attempt.", e);
      reportFailure(connectionUpdaterInput, null);
      continueAsNew(connectionUpdaterInput);
    }
  }

  private void generateSyncWorkflow(final ConnectionUpdaterInput connectionUpdaterInput) {
    syncWorkflow = Workflow.newCancellationScope(() -> {
      connectionId = connectionUpdaterInput.getConnectionId();

      if (connectionUpdaterInput.getWorkflowState() != null) {
        workflowState = connectionUpdaterInput.getWorkflowState();
      }

      workflowState.setResetConnection(connectionUpdaterInput.isResetConnection());

      final Duration timeToWait = getTimeToWait(connectionUpdaterInput.getConnectionId());
      Workflow.await(timeToWait, () -> skipScheduling() || connectionUpdaterInput.isFromFailure());

      if (needToProcessSync()) {
        jobId = getOrCreateJobId(connectionUpdaterInput);
        attemptId = createAttemptId(jobId);

        final GeneratedJobInput jobInputs = getJobInput();

        reportJobStarting();
        StandardSyncOutput standardSyncOutput = null;
        try {
          standardSyncOutput = runChildWorkflow(jobInputs);
          setFailStatus(standardSyncOutput);

          if (workflowState.isFailed()) {
            reportFailure(connectionUpdaterInput, standardSyncOutput);
          } else {
            reportSuccess(connectionUpdaterInput, standardSyncOutput);
          }

          continueAsNew(connectionUpdaterInput);
        } catch (final ChildWorkflowFailure childWorkflowFailure) {
          if (childWorkflowFailure.getCause() instanceof CanceledFailure) {
            // do nothing, cancellation handled by cancellationScope
            return;
          }

          if (childWorkflowFailure.getCause() instanceof final ActivityFailure af) {
            failures.add(FailureHelper.failureReasonFromWorkflowAndActivity(
                childWorkflowFailure.getWorkflowType(),
                af.getActivityType(),
                af.getCause(),
                jobId,
                attemptId));
          } else {
            failures.add(
                FailureHelper.unknownOriginFailure(childWorkflowFailure.getCause(), jobId, attemptId));
          }
          reportFailure(connectionUpdaterInput, standardSyncOutput);
        }
      }
    });
  }

Reply:
> can we rename `syncWorkflowCancellationScope` to `syncWorkflow`? I understand why the original name was called, however my initial read of it is this defines cancellation behaviour, and I had to read the constructor to understand what is going on.

I don't think that `syncWorkflow` would be a good idea because it is conflicting with the interface `SyncWorkflow`. I will rename it with something less confusing.

Reply:
Done

Reply:
Done